Add Formatting

JKTUNING opened the pull request:
## This PR add formatting standards to the project to help with maintainability, along with a few other additions.
- Add formatting with prettier
  - Add prettierrc.json and ignore files
 
- Remove Sleep package and use shell.exec('sleep x') instead
- Fix config file creation if file does not exist

- Changed the shell.exec command to use the full path to the binary (/usr/local/bin/flux-cli) to fix issues where flux-cli is returning an error.
    - Added error handling and error logging to better evaluate this behavior.
    - Future plans to try flux-cli/fluxbenchmark-cli and if this fails then fallback to trying to use the full path to the binary.

Cabecinha84 commented on watchdog.js:
```diff
@@ -165,75 +152,71 @@ return;
        await send_telegram_msg(emoji_title,info_type,field_type,msg_text,label);*/
 
-       sleep.sleep(3);
+      shell.exec("sleep 3");
```
I am not sure If I am a big fun of thos change.
I think I prefered old sleep lib, instead of calling shell to sleep :)

Cabecinha84 reviewed (approved):
Ack

Cabecinha84 commented:
Version needs to be bumped before it's merged.
@XK4MiLX @JKTUNING

JKTUNING commented:
I added the sleep package back and changed all sleep commands to use it, instead of shell.exec

I had also discussed with @XK4MiLX, about removing KDA checks from watchdog so those functions are now removed, along with unused packages and variables.

Version is not bumped yet.

Cabecinha84 commented:
If the kda checks works they should remain, as we still run some kda infrastructure that is used on Zelcore.

XK4MiLX commented:
> If the kda checks works they should remain, as we still run some kda infrastructure that is used on Zelcore.

That part of code do nathing continer name as diff and chainwebdata using separate component 
problem with kadena node freezing was "youth problems" + fdm have build own check

Cabecinha84 commented:
@XK4MiLX what is the state of this PR? Can the version be bumped and after merged?
